src/protocols/aeternity/components/TransferReview.vue

```diff
@@ -80,7 +80,7 @@
         <template #value>
           <TokenAmount
```

Comment:
Maybe we should use something other than `TokenAmount` here as the value is just a number and not an asset